cryptoki/src/session/object_management.rs

```diff
@@ -96,6 +96,40 @@ impl Session {
         }
     }
 
+    /// Copy an object
+    /// A template can be provided to change some attributes of the new object, when allowed.
```

Comment:
Please insert a new line here:

```suggestion
    ///
    /// A template can be provided to change some attributes of the new object, when allowed.
```